lib/components/Tooltip.tsx

```diff
@@ -0,0 +1,122 @@
+import type { CSSProperties, PropsWithChildren, ReactNode } from 'react'
+import { useState } from 'react'
+import { tx } from '@twind/core'
+
+type UseHoverStateProps = { closingDelay: number }
+const defaultUseHoverStateProps: UseHoverStateProps = {
+  closingDelay: 200
+}
+
+// TODO move this outside of this component and also replace inside the menu component
```

Comment:
Feel free to tackle this TODO in this PR

Reply:
Will be done in #1153